src/main/java/io/swagger/configuration/APIServiceConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package io.swagger.configuration;
+
+import io.swagger.entity.media.MediaInfo;
+import io.swagger.repository.MediaRepository;
+import io.swagger.service.MediaService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+
+@Configuration
+public class APIServiceConfiguration {
```

Comment:
This is the configuration file where we can start putting all beans (I didn't see a better file for this).

Reply:
You can actually just annotate your MediaService as a `@Service` and Spring will pick it it up as a bean and inject it

Reply:
I'll play around with @Service. I was wondering how your auth service was being injected since I couldn't find the bean. I'll make the change on the next PR I make.